GiniVision/Classes/Core/DocumentPickerCoordinator.swift

```diff
@@ -28,24 +34,70 @@ public protocol DocumentPickerCoordinatorDelegate: class {
                         didPick documents: [GiniVisionDocument])
 }
 
+/**
+ Document picker types.
```

Comment:
Isn't this comment redundant. It exactly matches the code below.

Reply:
No, because when they look up the class they don't see anything.